cw-orch/src/prelude.rs

```diff
@@ -27,8 +27,9 @@ pub use crate::environment::IndexResponse;
 
 // Environment
 pub use crate::environment::{
-    BankQuerier, BankSetter, CwEnv, DefaultQueriers, EnvironmentInfo, EnvironmentQuerier,
-    NodeQuerier, QuerierGetter, QueryHandler, TxHandler, TxResponse, WasmQuerier,
+    AccessConfig, BankQuerier, BankSetter, CwEnv, DefaultQueriers, EnvironmentInfo,
```

Comment:
I don't think AccessConfig should be in predule.
```suggestion
    BankQuerier, BankSetter, CwEnv, DefaultQueriers, EnvironmentInfo,
```